Jenkinsfile

```diff
@@ -182,7 +182,12 @@ def cacheName = "cache_${BUILD_TAG}"
 
 def testPrefix = "tinymce_${cleanBuildName(env.BRANCH_NAME)}-build${env.BUILD_NUMBER}"
 
-timestamps {
+timestamps { alertWorseResult(
+  cleanupStep: { devPods.cleanUpPod(build: cacheName) },
+  branches: ['main', 'release/7', 'release/8'],
```

Comment:
Can we do `release/*`? I'm tempted to suggest `epic/*` as well

Reply:
I ended up using exact string match for branch filtering since it could be confusing if people didn't know if it was regex/glob pattern. I did think about the `epic` case but I figured that would benefit more from using a different channel for notifications (since that one is more prone to failing) to avoid making too much noise in the `build-status` one. Otherwise it will get ignored, I guarantee it